MID Server/MID Server Script Includes/IllumioHTTPClient.js

```diff
@@ -105,15 +107,22 @@ IllumioHTTPClient.prototype = {
                 };
 
                 return responseObj;
-            } else {
-
+            }
+            if (retryCount == 0 || (RETRY_CODES.indexOf(responseCode) == -1 && !(responseCode >= 500))) {
                 return {
                     hasError: true,
                     status: responseCode
                 };
             }
+
+            lastSleepTime = lastSleepTime ? lastSleepTime : 0;
```

Comment:
ServiceNow currently doesn't support ES6 prior to Tokyo because of which we cannot give default parameters to this function yet as it is not supported in ES5

Reply:
Makes sense. This is fine until we can switch to ES6+.